GetPet/GetPet.Data/Entities/Pet.cs

```diff
@@ -20,7 +20,7 @@ public class Pet : BaseEntity
         public AnimalType AnimalType { get; set; }
                         
         [ForeignKey("User")]
-        public int UserId { get; set; }
+        public int? UserId { get; set; }
```

Comment:
don't break connections, 
use a default user (in the seed) for "מערכת" that insert crawled animals